db/wal_manager.h

```diff
@@ -47,7 +48,9 @@ class WalManager {
         seq_per_batch_(seq_per_batch),
         wal_dir_(db_options_.GetWalDir()),
         wal_in_db_path_(db_options_.IsWalDirSameAsDBPath()),
-        io_tracer_(io_tracer) {}
+        io_tracer_(io_tracer),
+        transaction_log_seq_cache_(
+            std::make_shared<TransactionLogSeqCache>(128, db_options_.clock)) {}
```

Comment:
nit: An inline comment for the purpose of 128 will be helpful for reading the code.